dotnet-desktop-guide/net/wpf/documents/snippets/how-to-print-xps-files/csharp/MainWindow.xaml.cs

```diff
@@ -0,0 +1,134 @@
+﻿using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Printing;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Windows;
+
+namespace CodeSampleCsharp
+{
+    /// <summary>
+    /// Interaction logic for MainWindow.xaml.
+    /// </summary>
+    public partial class MainWindow : Window
+    {
+        public MainWindow()
+        {
+            InitializeComponent();
+
+            // Events.
+            btnPrint.Click += BtnPrint_Click;
```

Comment:
The button click event should **not** be set here

Reply:
Done.